var/spack/repos/builtin/packages/libgtop/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Libgtop(AutotoolsPackage):
+    """Contains the GNOME top libraries for collecting system monitoring data"""
+
+    homepage = "https://gitlab.gnome.org/GNOME/libgtop"
+    url = "https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz"
+
+    maintainers("teaguesterling")
+
+    license("GPLv2", checked_by="teaguesterling")
+
+    # can be uncommented once glib@2.80 is added
+    # version("2.41.3", sha256="775676df958e2ea2452f7568f28b2ea581063d312773dd5c0b7624c1b9b2da8c")
```

Comment:
This is still missing, we should either remove it, or add the relevant versions of glib in another PR (I see the one mentioned in the description has been closed)

Reply:
There's some unresolved issues with glib 2.80 that were pointed out to me with the other PR. I kept these in comments to make additions easier in the future. 

See #44323.